xprof/btx_timeline.cpp

```diff
@@ -5,11 +5,16 @@
 #include <iomanip>  // set precision
 #include <iostream> // stdcout
 #include <map>
-#include <optional>
+//#include <optional>
```

Comment:
if `optional` not required, just remove it

Reply:
Sure, will do that.